### PR TITLE
Add acceptance tests for deleting shared files and deleting shares

### PR DESCRIPTION
**Pull request by dpakach**
## Description

Add acceptance tests for deleting the shared files.

## Related Issue

- Part of #1614 

## How Has This Been Tested?

:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

File: tests/acceptance/helpers/occHelper.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+const { client } = require('nightwatch-api')
+const fetch = require('node-fetch')
+const httpHelper = require('../helpers/httpHelper')
+
+exports.runOcc = async function (args) {
+  const headers = httpHelper.createAuthHeader('admin')
+  const params = new URLSearchParams()
+  params.append('command', args.join(' '))
+  return fetch(client.globals.backend_url + '/ocs/v2.php/apps/testing/api/v1/occ?format=json', {
```
Comment (individual-it) [inline tests/acceptance/helpers/occHelper.js]: check the response for correct status code

File: tests/acceptance/stepDefinitions/filesContext.js [hunk 2]
```diff
@@ -503,3 +503,8 @@ Then('the clipboard content should match permalink of resource {string}', functi
 Then('the thumbnail should be visible in the app-sidebar', function () {
   return client.page.FilesPageElement.appSideBar().isThumbnailVisible()
 })
+
+When('the user uploads overwriting file {string} using the webUI and retries if the file is locked', function (file) {
```
Comment (individual-it) [inline tests/acceptance/stepDefinitions/filesContext.js]: there is no "retry" when the file is locked. what happens when a file is locked in phoenix?

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 3]
```diff
@@ -70,66 +70,61 @@ Feature: Sharing files and folders with internal users
       | read                | Custom role | Viewer        | read                      |
       | all                 | Custom role | Editor        | all                       |
 
-  @skip @yetToImplement
   Scenario: share a file with another internal user who overwrites and unshares the file
     Given user "user2" has logged in using the webUI
     When the user renames file "lorem.txt" to "new-lorem.txt" using the webUI
-    And the user shares file "new-lorem.txt" with user "User One" using the webUI
+    And the user shares file "new-lorem.txt" with user "User One" as "Editor" using the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature]: we have sharing tests higher up, lets do this sharing as a Given step in the API

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUIFiles</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5472/

<p>






</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUISharingInternalUsers</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5472/

<p>





</p></details>

**Comment by ownclouders** [comment]
<details><summary>:boom: Acceptance tests <strong>webUITrashbin</strong> failed. Please find the screenshots inside ...</summary>

https://drone.owncloud.com/owncloud/phoenix/5600/

<p>



















</p></details>

File: tests/acceptance/pageObjects/filesPage.js [hunk 4]
```diff
@@ -163,6 +167,11 @@ module.exports = {
       return this.getAttribute('@sidebarItemName', 'innerText', function (itemName) {
         this.assert.strictEqual(itemName.value, resourceName, `In sidebar is different item - ${itemName.value}`)
       })
+    },
+    confirmFileOverwrite: function () {
+      return this
+        .waitForElementVisible('@fileOverwriteConfirm')
+        .click('@fileOverwriteConfirm')
```
Comment (individual-it) [inline tests/acceptance/pageObjects/filesPage.js]:
```suggestion
        .click('@fileOverwriteConfirm')
        .waitForElementNotVisible('@fileOverwriteConfirm')
        .waitForAjaxCallsToFinish()
```

File: tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature [hunk 5]
```diff
@@ -70,66 +70,61 @@ Feature: Sharing files and folders with internal users
...
+    When the user re-logs in as "user1" using the webUI
+    Then as "user1" the content of "new-lorem.txt" should not be the same as the local "new-lorem.txt"
     # overwrite the received shared file
-    When the user uploads overwriting file "new-lorem.txt" using the webUI and retries if the file is locked
+    When the user uploads overwriting file "new-lorem.txt" using the webUI
     Then file "new-lorem.txt" should be listed on the webUI
-    And the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
+    And as "user1" the content of "new-lorem.txt" should be the same as the local "new-lorem.txt"
     # unshare the received shared file
     When the user unshares file "new-lorem.txt" using the webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature]: there is no `unshare` button in phoenix, so lets use `delete` here

**Pull request merged**